tests/utils.go

```diff
@@ -760,6 +763,10 @@ func NewRandomVMWithEphemeralDiskAndUserdataHighMemory(containerImage string, us
 func NewRandomVMWithEphemeralDisk(containerImage string) *v1.VirtualMachine {
 	vm := NewRandomVM()
 
+	if strings.Contains(containerImage, "winrmcli") {
```

Comment:
no need, `winrmcli` container runs straightforward on a pod and not under VM
can you add this annotation under `windows_test.go` to `windowsVmSpec`